e2e/capacity/transactions.test.ts

```diff
@@ -49,7 +49,7 @@ describe('Capacity Transactions', function () {
   describe('pay_with_capacity', function () {
     describe('when caller has a Capacity account', function () {
       let schemaId: u16;
-      const amountStaked = 2n * DOLLARS;
+      const amountStaked = 3n * DOLLARS;
```

Comment:
This fixes the failing e2e tests. The weight change pushed something over that was only on the edge of working.